conda-envs/mlos-windows.yml

```diff
@@ -16,14 +16,14 @@ dependencies:
   - jupyter
   - ipykernel
   - nb_conda_kernels
-  - matplotlib
+  - matplotlib<2.9
+  - matplotlib-base<2.9
```

Comment:
I think this might need to be 3.9, not 2.9 since that's what's in `mlos_viz/setup.py`

Anyways, hopefully goes away once dabl cuts a new version with my fixes there.

Reply:
ooh right